### ISSUE TITLE
Where are the docs? The current example on the README.md doesn't work for me.

### ISSUE THREAD

**Issue by ZackPlauche** [googlemaps/google-maps-services-python#531]
**Is your feature request related to a problem? Please describe.**
I want to see how to use the API... current docs don't work.

**Comment by RavinduTharanga** [comment]
Did you try this? https://pypi.org/project/googlemaps/

**Comment by emigre459** [comment]
Yeah unfortunately I'm also finding the [straightforward example](https://github.com/googleapis/google-cloud-python/blob/main/packages/google-maps-routing/samples/generated_samples/routes_v2_generated_routes_compute_routes_sync.py) for routing_v2 errors out when you try to run it. Seems to be missing a number of args and kwargs to make it work (and I haven't been able to suss out exactly what's missing yet or I'd be posting an update to it).

**Comment by wescpy** [comment]
The root cause is that the current Google Maps client library is for the previous generation of Maps (Web Services) APIs, and the specific code sample that the OP is running only works with the new auto-generated client libraries for which there's no current documentation because it's so new. The team has yet to be able to document their use publicly.

The new Routes APIs don't work with the [current/older client libs](https://pypi.org/project/googlemaps). Google has stated that [none of the newer APIs will be supported by that library](https://github.com/googlemaps/google-maps-services-python/issues/509#issuecomment-1839113417). I don't know if the Maps team has linked to the new docs for the Routes v2 API yet, but you can [find them here](https://googleapis.dev/python/routing). If you set it up, the code sample _should_ run. Good luck!

ps. I'm slowly documenting [how to use Maps APIs from my tech blog](https://dev.to/wescpy/series/29655) and plan on getting to the new client libraries "soon," so keep an out there or to learn how to use [other Google APIs/platforms](https://dev.to/wescpy) (from Python or Node.js).
